README.md

```diff
@@ -23,4 +23,39 @@ The crawler can be started using the start script:
 
 ```shell script
 ./scripts/start.sh
-```
\ No newline at end of file
+```
+## Note
+
+Ensure your server has at least 3GB of memory as the Elasticsearch stack docker will require 2GB.
```

Comment:
Can you remove `your server`?

Ensure **you have** at least 3GB of memory as ...